integration-tests/tests/src/tests/sync/flexible.ts

```diff
@@ -268,37 +268,34 @@ describe.skipIf(environment.missingServer, "Flexible sync", function () {
 
           async function openRealm(user: Realm.User, extraConfig: ExtraConfig = {}) {
             const config = getSuccessConfig(user, extraConfig);
-            const realm = await Realm.open(config);
-
-            return { realm, config };
+            return Realm.open(config);
```

Comment:
could you elaborate to me why `await` no longer is needed for opening this realm? Should the function still be async if that's no longer the case?

Reply:
i'll chime in.  The `await` here was redundant.  The code consuming this function assumes a promise anyway and would `await` the result.

Reply:
I see, appreciate the explanation! :D